register-system.yaml

```diff
@@ -11,6 +11,20 @@
         packages:
           - dmidecode
           - ipmitool
+          - lldpad
+
+    - name: "Start lldpad service"
+      ansible.builtin.systemd:
+        name: lldpad
+        state: started
```

Comment:
We're not setting `enabled: true` on this service, so it won't start up next time the system boots. Is that intentional? If so, do we want to update the playbook to *stop* the service as well? We can do that with a `block` and an `always` clause (sort of like `try`/`finally` in Python):

```
- block:
    - name: Start lldpad service
      ansible.builtin.systemd:
        name: lldpad
        state: started

    # ...other tasks here...
  always:
    - name: Stop lldpad service
      ansible.builtin.systemd:
        name: lldpad
        state: stoped
```

Reply:
I think I will set enabled true since if some switchport changes happen then those will be updated.